apps/challenges/models.py

```diff
@@ -172,6 +172,8 @@ def __init__(self, *args, **kwargs):
     desired_worker_instance = models.IntegerField(
         null=True, blank=True, default=1
     )
+    # Allow ordering leaderboard by all metrics
+    is_multi_metric_leaderboard = models.BooleanField(default=False)
```

Comment:
Since we have the `metadata` field in the database, I don't think we will need this field.